dashboard/app/controllers/script_levels_controller.rb

```diff
@@ -246,6 +246,11 @@ def present_level
       script_level_id: @script_level.id,
       level_id: @level.id)
 
+    @sublevel_callback = milestone_script_level_url(
+      user_id: current_user.try(:id) || 0,
+      script_level_id: @script_level.id,
+      level_id: '') if @level.game.level_group?
```

Comment:
Sorry that I didn't comment on this one. I think the "if at the end" thing is common in ruby. I think it makes sense to use if you have one statement (not necessarily one line). Perhaps this is covered in code.org style guide?

I don't know if there's any advantage to not using it just because the parameters wrapped. I guess its a tradeoff in readability (noticing the "if", which is always an issue for someone coming from non-ruby languages like me) vs. manageability (extra long files and indentation for simply if / end blocks with only one statement inside.